diff --git a/vendor/gaeutilities/cache.py b/vendor/gaeutilities/cache.py
index ec4916a..07245fa 100644
--- a/vendor/gaeutilities/cache.py
+++ b/vendor/gaeutilities/cache.py
@@ -38,9 +38,9 @@ from google.appengine.api import memcache
 
 # settings
 try:
-    import settings
+    from vendor.gaeutilities import settings
 except:
-    import settings_default as settings
+    from vendor.gaeutilities import settings_default as settings
     
 class _AppEngineUtilities_Cache(db.Model):
     cachekey = db.StringProperty()
diff --git a/vendor/gaeutilities/flash.py b/vendor/gaeutilities/flash.py
index 2dffd6c..3401d14 100644
--- a/vendor/gaeutilities/flash.py
+++ b/vendor/gaeutilities/flash.py
@@ -33,9 +33,9 @@ from django.utils import simplejson
 
 # settings
 try:
-    import settings
+    from vendor.gaeutilities import settings
 except:
-    import settings_default as settings
+    from vendor.gaeutilities import settings_default as settings
 
 COOKIE_NAME = settings.flash["COOKIE_NAME"]
 
diff --git a/vendor/gaeutilities/rotmodel.py b/vendor/gaeutilities/rotmodel.py
index 4084094..e0a5860 100644
--- a/vendor/gaeutilities/rotmodel.py
+++ b/vendor/gaeutilities/rotmodel.py
@@ -31,9 +31,9 @@ from google.appengine.ext import db
 
 # settings
 try:
-    import settings
+    from vendor.gaeutilities import settings
 except:
-    import settings_default as settings
+    from vendor.gaeutilities import settings_default as settings
 
 class ROTModel(db.Model):
     """
diff --git a/vendor/gaeutilities/sessions.py b/vendor/gaeutilities/sessions.py
index 191dd11..94a897c 100644
--- a/vendor/gaeutilities/sessions.py
+++ b/vendor/gaeutilities/sessions.py
@@ -48,9 +48,9 @@ from rotmodel import ROTModel
 
 # settings
 try:
-    import settings
+    from vendor.gaeutilities import settings
 except:
-    import settings_default as settings
+    from vendor.gaeutilities import settings_default as settings
 
 
 class _AppEngineUtilities_Session(ROTModel):
